components/threading/common/button/button.tsx

```diff
@@ -35,7 +37,7 @@ function Button({
                     {prepend}
                 </span>
             )}
-            <span className='Button_label'>
+            <span className={marginExtra ? 'Button_label_margin_extra' : 'Button_label'}>
```

Comment:
after the above change we can just use `className('Button_label, {'margin_extra': marginExtra}`